mobx_codegen/lib/src/template/store.dart

```diff
@@ -56,6 +60,25 @@ abstract class StoreTemplate {
 
   $actions''';
 
+    if (generateToString) {
+      toStringList
+        ..addAll(observables.templates.map(
+            (current) => '${current.name}: \${${current.name}.toString()}'))
+        ..addAll(computeds.templates.map(
+            (current) => '${current.name}: \${${current.name}.toString()}'));
```

Comment:
We should filter to only include fields that are **public**.

Reply:
you mean removing all fields with name starting with '_'  ?

Reply:
Yes, but the check should be on the element, not the field name. I think you can check the access level of the element (`isPublic`??) and then filter.

Reply:
At this point it's too late to use isPublic.
I can check isPublic when we add elements to observables list and computed list. At this point I can either mark the element as public in its template or add it to a specific toString list inside the StoreTemplate.

Reply:
no worries, take your time. Yes, we should test before adding the fields to the list. We can check for `isPublic` before adding. The PR is shaping up well. Great job so far! We are close to the finish line